projects/example-app/karma.conf.js

```diff
@@ -0,0 +1,31 @@
+// Karma configuration file, see link for more information
```

Comment:
Is the karma config needed? (because we're using Jest)

Reply:
I'm gonna leave it for now because it makes upgrading with the CLI easier